src/motivation.md

```diff
@@ -0,0 +1,48 @@
+# Motivation
+
+The decentralized internet demands privacy-respecting data provenance!
```

Comment:
imo, not many know what "data provenance" means 
Im in favor of avoiding "provenance" altogether in this section and use "privacy-respecting proof of data authenticity" instead.
But if using "provenance" cannot be avoided, at least we could define what provenance is at the top of this write-up.

Reply:
As far as I know, 'data provenance' is the best technical term. That's why I would use it here as well. However, it's indeed a good idea to provide a brief explanation of the term here.